recipes/vivid/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "vivid" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sharkdp/vivid/archive/v{{ version }}.tar.gz
+  sha256: 325f16df916e4192d56d589839be474801ffd3d6a105f4a1a033221570ff6b56
+
+build:
+  number: 0
+  ignore_run_exports:
+    - libstdcxx-ng
```

Comment:
Why have you added this?

Reply:
This indicates that you don't actually need the C++ compiler.

Reply:
Ignorance, I guess.  I copied a working recipe.  Should I remove it?